src/app/games/lasca/lasca.component.html

```diff
@@ -0,0 +1,48 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+     width="100%" height="100%"
+     [attr.viewBox]="LEFT + ' ' + UP + ' ' + WIDTH + ' ' + HEIGHT"
+     preserveAspectRatio="xMidYMid meet">
+    <defs>
+        <g id="piece">
+            <path d="M 10 15 v 15 a 40 15, 0, 0, 0, 80 0 v -15 a 40 15, 0, 0, 0, -80 0 Z"/>
+            <ellipse cx="50" cy="15" rx="40" ry="15"/>
+        </g>
+    </defs>
+    <g *ngFor="let line of adaptedBoard; index as y">
+        <g *ngFor="let squareInfo of line; index as x">
+            <g id="coord_{{ x }}_{{ y }}"
+               (click)="onClick(x, y)"
+               [attr.transform]="getCoordTransform(x, y)">
+                <polygon id="square_{{ x }}_{{ y }}"
+                         [attr.points]="getRhombusPoints()"
+                         [ngClass]="squareInfo.squareClasses"
+                         class="base-no-fill"
+                         [attr.fill]="backgroundColor(x, y)"/>
+                <g *ngFor="let pieceInfo of squareInfo.pieceInfos; index as z"
+                    pointer-events="fill"
+                    [attr.transform]="getTransform(z)"
+                    class="base mid-small-stroke">
+                    <use id="square_{{ x }}_{{ y }}_piece_{{ z }}"
+                         x="25"
+                         y="35"
```

Comment:
kéé

Reply:
j'ai multiplié par space_size et divisé par 100 pour que ce soit plus refactorable.
Sinon euh, c'est un peu du sorti manuellement pour que la pièce cylindrique soit centrée dans la case rhombique, que préconise tu ?

Reply:
de remplacer par SPACE_SIZE*???? / 100 si c'est le calcul que t'as fait (mais j'ai pas compris ce que tu as multiplié)
Histoire qu'on explicite les points variables (space size dans ce cas, voire un autre)

Reply:
Euh, out-of-date, maintenant c'est plus clair